indico/modules/designer/controllers.py

```diff
@@ -190,6 +203,7 @@ def clone_template(self, target=None):
         new_template = DesignerTemplate(title=title, type=self.template.type, **target_dict)
 
         data = deepcopy(self.template.data)
+        data = remove_regform_field_placeholders(data)
```

Comment:
I'd rather clone it as linked. It's nice if you can have a backup or want to make just some minor changes!